### PR TITLE
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/1549

### PR DESCRIPTION
**Pull request by VLG17**
updating info regarding Windows Defender Security Center

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dansimp Would you be able to review and merge this PR as time permits? Thank you.

File: windows/security/threat-protection/windows-defender-security-center/wdsc-hide-notifications.md [hunk 1]
```diff
@@ -30,6 +30,9 @@ manager: dansimp
 
 - Group Policy
 
+>[!NOTE]
+>In Windows 10, version 1803 and above, the Security app is referred to as **Windows Defender Security Center** and has two new areas, **Account protection** and **Device security**.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-security-center/wdsc-hide-notifications.md]:
```suggestion
>In Windows 10, version 1803 and above, the Security app is referred to as **Windows Defender Security Center** and has two new areas; **Account protection** and **Device security**. 
```

**Comment by e0i** [comment]
@dansimp Any updates on this? Thanks.

**Comment by VLG17** [comment]
as discussed with @mypil

**Pull request closed without merging**